package.json

```diff
@@ -28,7 +28,11 @@
     "security-checks": "cross-env NODE_PATH=. grunt security-checks",
     "healthcheck": "cross-env NODE_PATH=. mocha test/healthCheck.js --timeout 15000 --reporter spec",
     "post-data": "cross-env NODE_PATH=. node post-data.js",
-    "sonar-scan": "sonar-scanner"
+    "sonar-scan": "node_modules/sonar-scanner/bin/sonar-scanner",
```

Comment:
Is this full path required? Looking in the `node_modules/bin` directory I can see the `sonar-scanner` binary, anything living in there is automatically picked up. In short, `"sonar-scan": "sonar-scanner"` should be suffice.